How to/Library Bounds to Viewer bounds/index.js

```diff
@@ -0,0 +1,92 @@
+var pdfviewer = new ej.pdfviewer.PdfViewer({
+  documentPath: 'https://cdn.syncfusion.com/content/pdf/pdf-succinctly.pdf',
+  serviceUrl: 'https://services.syncfusion.com/js/production/api/pdfviewer'
+});
+ej.pdfviewer.PdfViewer.Inject(ej.pdfviewer.TextSelection, ej.pdfviewer.TextSearch, ej.pdfviewer.Print, ej.pdfviewer.Navigation, ej.pdfviewer.Toolbar,
+                            ej.pdfviewer.Magnification, ej.pdfviewer.Annotation, ej.pdfviewer.FormDesigner, ej.pdfviewer.FormFields, ej.pdfviewer.PageOrganizer);
+pdfviewer.appendTo('#PdfViewer');                              
+var pageSizes = [];
+pdfviewer.ajaxRequestSuccess = function (args) {
+  if (args.action === 'Load') {
+    let objLength = Object.keys(args.data.pageSizes).length;
+    for (var x = 0; x < objLength; x++) {
+      var pageSize = args.data.pageSizes[x];
```

Comment:
@SF4524LogeshKumar  - Please use the getPageInfo API to retrieve the page size instead of ajaxRequestSuccess event.

Reply:
Addressed Suggestion. Implemented code changes and optimized code.